Update covid-19 page with postponing details (#204)
srobo/website: covid-19.md

## The Competition

Following guidance from the UK government and the University of Reading, **we will need to postpone Student Robotics 2020** from the original date in April, until some time late June / early July.

> We have sent a poll to team leaders and volunteers to allow us to select the most appropriate date.

Reviewer: ```suggestion
We have requested feedback from team leaders and volunteers to help select the most appropriate date.
```

### During the competition

We will be briefing all volunteers and attendees about best hygiene practices, both in advance and on the day, ensuring they're followed throughout the competition.

We expect everyone attending the competition to follow the [guidelines][phe-guidelines] on COVID-19 from Public Health England.

## Tech Days

> There are no current planned tech days before the current competition date in April. Once things have settled down, we hope to organise some.

Reviewer 2: The planned date is no longer April.
